src/decode.h

```diff
@@ -1015,7 +1018,7 @@ void DecodeUnregisterCounters(void);
 #define PKT_STREAM_NOPCAPLOG BIT_U32(12)
 
 #define PKT_TUNNEL           BIT_U32(13)
-#define PKT_TUNNEL_VERDICTED BIT_U32(14)
+// vacancy
 
 /** Packet checksum is not computed (TX packet for example) */
 #define PKT_IGNORE_CHECKSUM BIT_U32(15)
```

Comment:
Can't all the defines be shifted by two?

Reply:
Not shifting to keep flags stable. This mostly helps in debugging reported core dumps.